lib/order.rb

```diff
@@ -0,0 +1,26 @@
+class Order 
```

Comment:
Name of variables are quite adequate and easy to understand what the code is trying to do